components/actions/actionsSection.tsx

```diff
@@ -48,11 +52,19 @@ export default function PageActionsSection(props: {
   );
 
   useEffect(() => {
-    setNumActiveActions(
-      props.actionTags
-        .map((tag) => tag.actions)
-        .flat()
-        .filter((action) => action.active).length,
+    const activeActions = props.actionTags
+      .map((tag) => tag.actions)
+      .flat()
+      .filter((action) => action.active);
+    setNumActiveActions(activeActions.length);
+
+    const tokenCount = getTokenCount([
+      { role: "system", content: getActionDescriptions(activeActions) },
```

Comment:
This isn't the full system prompt, right? So you could still have this issue, since there's a fair amount of other instruction in the prompt